README.md

```diff
@@ -9,21 +9,38 @@ Fleet Telemetry is a server reference implementation for Tesla's telemetry proto
 
 The service handles device connectivity as well as receiving and storing transmitted data. Once configured, devices establish a WebSocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
 
-## Configuring and running the service
+## Requirements
+These are the minimum system requirements for a Fleet Telemetry server, with 10 active vehicles connected, running on a Debian-based server with regular day-to-day usage. The requirements will change based on vehicle usage and quantity of vehicles.
```

Comment:
minor: the first comma here isn't grammatically correct.
Maybe reword to something like: `These minimum system requirements assume a server with: 10 active vehicles connected, running on a Debian-based image, and with regular day-to-day vehicle usage.`

Reply:
Why are we having 10 active vehicle as a reference? probably better to have 1k.